canonical-websites/snapcraft.io
# Open releases list in the overlay

Fixes #1267 

As part of visual updates for release UI adds an overlay with releases history for clicked release table cell.

### QA

- ./run or demo
- go to releases page of any snap
- click on 'Show available revisions' - revisions list should open below releases table
- click on any cell - release history should open in overlay below given table row

<img width="1032" alt="screen shot 2018-11-14 at 13 07 38" src="https://user-images.githubusercontent.com/83575/48481557-4df6b480-e80e-11e8-9262-64f88543f6c4.png">
